app/kuma-dp/cmd/run.go

```diff
@@ -71,7 +71,7 @@ func newRunCmd(rootCtx *RootContext) *cobra.Command {
 
 			proxyResource, err = readResource(cmd, &cfg.DataplaneRuntime)
 			if err != nil {
-				runLog.Error(err, "failed to read %s policy", cfg.Dataplane.ProxyType)
+				runLog.Error(err, "failed to read %s policy", "proxyType", cfg.Dataplane.ProxyType)
```

Comment:
Ugh. It would not be hard to write a linter that checks this, though I couldn't find an existing one.

Reply:
```suggestion
				runLog.Error(err, "failed to read policy", "proxyType", cfg.Dataplane.ProxyType)

```